### ISSUE TITLE
The prefix "m->" is not recognized in codeblocks

### ISSUE THREAD

**Issue by leon-ts** [X-Sharp/XSharpPublic#738]
The following code generates an error on the function Eval (argument error):
```
PUBLIC p_date
m->p_date := Today()

LOCAL cS AS STRING
cS := "{|| m->p_date }"
LOCAL cbCompiledS AS CODEBLOCK
cbCompiledS := &cS
LOCAL uResult AS USUAL
uResult := Eval(cbCompiledS)
```

If I remove the m-> prefix from the codeblock, then Eval is executed correctly.
Also, the _MEMVAR-> prefix does not work in the codeblock.

**Comment by RobertvanderHulst** [comment]
This is a problem in the __FieldGetWa() function. It does not detect that the area parameter is a SYMBOL and therefore does not see that it has the value "M".
Alternatively the macro compiler could pass "M" as string

**Comment by leon-ts** [comment]
Robert,

There is also the _MEMVAR keyword (the same as M). It also doesn't work in X# codeblocks.
For example:
```
PUBLIC p_date
_MEMVAR->p_date := Today()
	
cS := "{|| _MEMVAR->p_date }"
cbCompiledS := &cS
uResult := Eval(cbCompiledS)
```

**Comment by RobertvanderHulst** [comment]
I Will look at memvar-> too

**Comment by RobertvanderHulst** [comment]
Fixed in c99c2ebc9e0bafba3ad923ab2f8aaf65308fe4c8

**Comment by leon-ts** [comment]
Robert,
Just to be sure, the **_MEMVAR** keyword used in the expressions has an underscore at the beginning of the word.

**Comment by cpyrgas** [comment]
Confirmed fixed.

**Comment by leon-ts** [comment]
Guys,
Is it realistic to patch this fix for version 2.8c? This refers to those problems in the code that are very difficult to overcome (in a large application). I'm also interested in fixing _SIZEOF: https://github.com/X-Sharp/XSharpPublic/issues/735
